index.md

```diff
@@ -54,7 +54,7 @@ eventbrite:           # optional: clave alfanumérica de registro en Eventbrite,
 {% comment %}
   INTRODUCCIÓN 
 
-  Edita el párrafo introductorio general debajo si quieres modificar la presentación.
+  Edita el párrafo introductorio general debajo si quieres modificar la presentación. Asegúrate de modificar el texto debajo del tipo de taller que estás organizando (swc, dc o lc).
```

Comment:
I forgot to remove this because I later noticed that we actually just need to change the `intro.html` file. I do think it is worth explaining that this, especially for people (like myself) who are new to these sort of files and may not notice this the first time. Should we include something like:
_Para editar el párrafo introductorio general deberás modficar el archivo_ `intro.html`_. Este archivo lo puedes encontrar navegando a la carpeta_ `includes` _y luego selecciona la carpeta para el tipo de taller que estás organizando (sw, dc, o lc)._
¿Qué te parece el párrafo de arriba?

Reply:
Me parece perfecto! Cualquier cosa que mejore la claridad es bienvenida.